app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def new; end
```

Comment:
- [ ] Please delete any unused code for a cleaner and more optimized code base. 🙏